src/Main.cc

```diff
@@ -131,6 +133,11 @@ static void initPython()
     exit(1);
   }
 
+  if (PyImport_AppendInittab("_gpl_py", PyInit__gpl_py) == -1) {
+    fprintf(stderr, "Error: could not add module _gpl_py\n");
+    exit(1);
+  }
+
```

Comment:
I wonder if this could be a function or preprocessor define considering it's going to be made for every tool, e.g.

```
void pyInit(a, b) {
  if (PyImport_AppendInittab(a, b) == -1) {
    fprintf(stderr, "Error: could not add module %s\n", a);
    exit(1);
  }
}
```
or
```
define PYINIT(X) \
  if (PyImport_AppendInittab(X, PyInit__##X) == -1) { \
    fprintf(stderr, "Error: could not add module X\n"); \
    exit(1); \
  }
```
Same for any repeated code.

Reply:
Yes, I did consider that, but it's always easier to do "just one more". It does seem like the perfect example for metaprogramming, though. My heartache in general is that people get too frisky with it, but I don't think that will be the case here. In any event, that should be in a separate PR. I'll take a look after this one is merged.

Reply:
No worries, just a suggestion.

Reply:
So I have done this now in 2427. Take a look and let me now what you think.